src/main/java/UITestFramework/CreateSession.java

```diff
@@ -222,19 +220,19 @@ public void startAppiumServer(String os) throws ExecuteException, IOException, I
 
 		}
 		else if (os.contains("mac os x")){
-			CommandLine command = new CommandLine("/Applications/Appium.app/Contents/Resources/node/bin/node");  
-			command.addArgument("/Applications/Appium.app/Contents/Resources/node_modules/appium/bin/appium.js", false);  
-			command.addArgument("--address", false);  
-			command.addArgument("127.0.0.1");  
-			command.addArgument("--port", false);  
-			command.addArgument("4723");  
-			command.addArgument("--full-reset", false);  
-			DefaultExecuteResultHandler resultHandler = new DefaultExecuteResultHandler();  
-			DefaultExecutor executor = new DefaultExecutor();  
-			executor.setExitValue(1);  
-			executor.execute(command, resultHandler);  
-			Thread.sleep(5000);  
+			String appiumJSPath = "/Users/mac/.npm/lib/node_modules/appium/build/lib/main.js";
```

Comment:
Is the "appiumJSPath" property mandatory? I can start the server without it on my machine. Can you try on your end if it runs without this property?

Reply:
No, it's not required. I had checked with my code.